src/App.module.css

```diff
@@ -3,7 +3,6 @@
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  padding: 6rem;
```

Comment:
What does this do? Or no longer do?

Reply:
it removes the padding from the body. When the modal is changed the whatever padding is set on the body gets messed up. So i added it on an internal div as it should be